modules/ldap-auto-syncing.adoc

```diff
@@ -0,0 +1,221 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-auto-syncing_{context}"]
+= Automatically syncing LDAP groups
+
+You can automatically sync LDAP groups on a periodic basis by configuring a cron job.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have configured an LDAP identity provider (IDP).
++
+This procedure assumes that you created an LDAP secret named `ldap-secret` and a config map named `ca-config-map`.
+
+.Procedure
+
+. Create a project where the cron job will run:
++
+[source,terminal]
+----
+$ oc new-project ldap-sync <1>
+----
+<1> This procedure uses a project called `ldap-sync`.
+
+. Copy the secret and config map that you created when configuring the LDAP identity provider to this new project.
```

Comment:
In this construction, "...to this new project" seems to be hanging at the end. Consider rewording for clarity?
```suggestion
. Locate the secret and config map that you created when configuring the LDAP identity provider and copy them to this new project.
```

Reply:
Alternatively, you might split this into two steps, as that's sort of how it reads anyway:
1. Locate the secret and config map that you created when configuring the LDAP identity provider. These exist in the `openshift-config` project.
2. Copy the secret and config map to the new `ldap-sync project`.

Reply:
Good suggestion, thanks!